server/src/DBTablesAction.cc

```diff
@@ -948,6 +980,82 @@ bool DBTablesAction::getLog(ActionLog &actionLog, const string &condition)
 	return true;
 }
 
+// TODO: Extract commonly used lines in getLog() above and reduce the similar lines. 
+bool DBTablesAction::getLogs(ActionLogList &actionLogList,
+                             const string &condition)
+{
+	DBAgent::SelectExArg arg(tableProfileActionLogs);
+	arg.condition = condition;
+	arg.add(IDX_ACTION_LOGS_ACTION_LOG_ID);
+	arg.add(IDX_ACTION_LOGS_ACTION_ID);
+	arg.add(IDX_ACTION_LOGS_STATUS);
+	arg.add(IDX_ACTION_LOGS_STARTER_ID);
+	arg.add(IDX_ACTION_LOGS_QUEUING_TIME);
+	arg.add(IDX_ACTION_LOGS_START_TIME);
+	arg.add(IDX_ACTION_LOGS_END_TIME);
+	arg.add(IDX_ACTION_LOGS_EXEC_FAILURE_CODE);
+	arg.add(IDX_ACTION_LOGS_EXIT_CODE);
+	arg.add(IDX_ACTION_LOGS_SERVER_ID);
+	arg.add(IDX_ACTION_LOGS_EVENT_ID);
+
+	getDBAgent().runTransaction(arg);
+
+	const ItemGroupList &grpList = arg.dataTable->getItemGroupList();
+	ItemGroupListConstIterator itemGrpItr = grpList.begin();
```

Comment:
The following comment is just my impression.
No need to take care of it.

Since C++11, `cbegin` and `cend` members has been added in iterator member.
If we start to use `auto` in type to use value type inference, it must use to create `const_iterator`s.
But, we sometimes write type name explicitly. No need to use `cbegin()` here for now.